library/HTMLPurifier/Bootstrap.php

```diff
@@ -63,7 +63,14 @@ public static function getPath($class)
         // Custom implementations
         if (strncmp('HTMLPurifier_Language_', $class, 22) === 0) {
             $code = str_replace('_', '-', substr($class, 22));
+            // Classic structure
             $file = 'HTMLPurifier/Language/classes/' . $code . '.php';
+
+            // New PSR-0 compatible structure
+            $fileAlt = 'HTMLPurifier/Language/' . $code . '.php';
+
+            // Switch to PSR-0 compatible if possible
+            $file = file_exists(HTMLPURIFIER_PREFIX . '/' . $fileAlt) ? $fileAlt : $file;
```

Comment:
This *does* introduce some extra IO overhead by checking if the language file exists in the root directory, but returning early here makes this method wet. And since it's an autoloader, it's only run once per locale (which is usually just 1 per request)